tests/common/experimental/test_reducers_and_aggregators.py

```diff
@@ -254,38 +252,38 @@ def test_min_max_aggregators(
         assert self.all_close(max_aggregator.aggregate(), max_ref)
 
     NO_OUTLIERS_TEST_PARAMS = [
-        (MeanAggregator, True, 1, 1404.5138888888905),
-        (MedianAggregator, True, 1, 24.0),
+        (MeanAggregator, True, 1, [1404.5138888888905]),
+        (MedianAggregator, True, 1, [24.0]),
```

Comment:
Is it possible with the changes that scalar statistics stay as scalar?

Reply:
The logic should be the same for all tensors: to keep dims of the original tensor.
In that test input tensor has `dim=1`